docs/user-guide/continuous-aggregation/overview.md

```diff
@@ -133,12 +133,15 @@ Here is the explanation of the columns in the `ngx_statistics` table:
 - `time_window`: The time window of the aggregation.
 - `update_at`: The time when the aggregation is updated.
 
+<!-- TODO(discord9): improve auto create table then add back this feature explain, i.e. for now everything in group by is put to primary key, and time index is always a placeholder -->
```

Comment:
Will you add these to this PR?

Reply:
> Will you add these to this PR?

Probably not, since the recommended way to create table is still manually do it